.travis.yml

```diff
@@ -19,13 +19,18 @@ matrix:
         - php: hhvm
 
 before_install:
+    - if [ -f "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" ]; then mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini-disabled"; fi;
     - composer self-update
+    - if [ -n "$GITHUB_OAUTH_TOKEN" ]; then composer config --global github-oauth.github.com ${GITHUB_OAUTH_TOKEN}; fi;
```

Comment:
Yeah fore sure bit it just makes it more unreadable for no real reason: if the token isn't there I would rather have an error and fix it, especially for something that's not supposed to move at all is it?